applications/asset_tracker_v2/boards/nrf9160dk_nrf9160ns.conf

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Configuration file for nrf9169dk.
```

Comment:
s/nrf9169dk/nrf9160dk

but I recommend to remove this line as such comments are prone to be leftovers during copy/pasting